ui/packages/app/src/components/Dataset/ExampleComponents/Image.svelte

```diff
@@ -4,4 +4,7 @@
 </script>
 
 <!-- svelte-ignore a11y-missing-attribute -->
-<img class="gr-sample-image" src={samples_dir + value} />
+<img
+	class="gr-sample-image object-contain h-20 w-20"
```

Comment:
@aliabid94 does this change make sense? Since we only want to resize the example thumbnail image; and not actually modify the image itself.

Reply:
I think it makes sense, just test it with different image sizes. Also let's add these classes to the tokens.css file where all the other classes are applied to `gr-sample-image`, not in this file